R/pkg/R/DataFrame.R

```diff
@@ -3731,6 +3743,9 @@ setMethod("histogram",
 #'
 #' Save the content of the SparkDataFrame to an external database table via JDBC. Additional JDBC
 #' database connection properties can be set (...)
+#' You can find the JDBC-specific option and parameter documentation for writing tables via JDBC in
+#' \href{https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html#data-source-option}{
+#' Data Source Option} in the version you use.
```

Comment:
I wonder if we also need to to move this to the `param`.

Maybe I don't think  we should move this because the description of `param` is about "database connection properties", not about "data source option", though

Reply:
Yeah this one lets leave it as is.